api/android/api/src/androidTest/java/org/nnsuite/nnstreamer/APITestCustomFilter.java

```diff
@@ -279,13 +278,13 @@ public void testRegisterDuplicatedName() {
             CustomFilter.registerCustomFilter(mCustomPassthrough.getName(),
                 new CustomFilter.CustomFilterCallback() {
                     @Override
-                    public TensorsInfo getOutputInfo(TensorsInfo inInfo) {
-                        return inInfo;
+                    public Tensor getOutputInfo(Tensor in) {
+                        return in;
                     }
 
                     @Override
-                    public TensorsData invoke(TensorsData inData, TensorsInfo inInfo, TensorsInfo outInfo) {
-                        return inData;
+                    public Tensor invoke(Tensor in) {
```

Comment:
It seems that the changes are good to improve readability.

Reply:
PR changed (TensorsData class with metadata).